### PR TITLE
#160845490 Setup SASS processing on the client

### PR DESCRIPTION
**Pull request by DrKimpatrick**
#### What does this PR do?
- Installs and setup sass pre-processing on the application to enable the use of both .scss and .css files on the client
#### Description of Task to be completed?
- configure React app to support sass
- enable the use of both .css and .scss files on the client
#### How should this be manually tested?
* This feature can be manually tested by running the following 
- Run `npm install `
- Run `npm start`

#### What are the relevant pivotal tracker stories?
[#160845490](https://www.pivotaltracker.com/story/show/160845490)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 import App from './App';
+import './redux-js/index';
+import registerServiceWorker from './registerServiceWorker';
+
 ReactDOM.render(
-     <provider store={store}>
-         <App />
-     </provider>
-         , document.getElementById('root'));
+	<Provider store={window.store}>
```
Comment (Estaer) [inline src/index.js]: What is the relevance of adding `{window.store}` instead of maintaining `{store}`?

File: .eslintrc.json [hunk 2]
```diff
@@ -27,7 +37,7 @@
     "eqeqeq": [2, "allow-null"],
     "generator-star-spacing": [2, { "before": true, "after": true }],
     "handle-callback-err": [2, "^(err|error)$" ],
-    "indent": [2, 2, { "SwitchCase": 1 }],
+    "indent": [2, "tab", { "SwitchCase": 1 }],
```
Comment (collinewait) [inline .eslintrc.json]: What is the relevance of `2` and `{ "SwitchCase": 1 }`  instead of just `"tab"`?

File: src/index.js (hunk 1 above)
Comment (DrKimpatrick) [reply src/index.js]: By using window.store in the place of `store` makes the store global

File: .eslintrc.json (hunk 2 above)
Comment (collinewait) [reply .eslintrc.json]: Cool.

**Comment by collinewait** [comment]
LGTM

File: src/index.js (hunk 1 above)
Comment (Estaer) [reply src/index.js]: Okay then. 
LGTM

**Pull request merged**